Move call for proposal result page to cfp folder

railsstudent opened the pull request:
1) Move call for proposal results page to cfp folder
2) Change results property in yml file to topics

railsstudent committed:
Move call for proposal result page to cfp folder

tonyhhyip commented on assets/layouts/navbar.jinja:
```diff
@@ -28,7 +28,7 @@
                 <li><a href="{{ data.site.basePath }}/about/staff.html">Staff</a></li>
                 <li><a href="{{ data.site.basePath }}/about/patron-programme.html">Patron Programme</a></li>
-                <li><a href="{{ data.site.basePath }}/about/cfp.html">Call For Proposals Results</a></li>
+                <li><a href="{{ data.site.basePath }}/cfp/cfp.html">Call For Proposals Results</a></li>
```
Use `{{ data.site.basePath }}/cfp/`

railsstudent committed:
Rename file to index.jinja

1) Change css class of the jinja file
2) Move CFP Result page out of About to a new menu item.
3) Change url in navbar html
